lib/OpenQA/WebAPI/Plugin/ObsRsync/Task.pm

```diff
@@ -52,7 +52,7 @@ sub run {
     return $job->retry({delay => $retry_interval})
       unless my $concurrency_guard = $minion->guard('obs_rsync_run_guard', $lock_timeout, {limit => $concurrency});
 
-    my @cmd = ('bash', "$home/rsync.sh", $project);
+    my @cmd = ("$home/script/rsync.sh", $project);
```

Comment:
Side note: path building should generally be done by a module that knows how to deal with portability issues, like `Mojo::File`.
```perl
my @cmd = (Mojo::File->new($home, 'script', 'rsync.sh')->to_string, $project);
```

Reply:
good idea to change this while we are on it. After change we can merge and have it effective on o3 for experimentation and keep it in mind for the next osd deployment to sync with the according changes there.